_docs/paste-lab.md

```diff
@@ -25,6 +25,26 @@ The objects are now pasted to fill the entire slide.
 
 Multiple objects can be selected to be used for Paste To Fill Slide, and the pasted objects will automatically be grouped together.
 
+## <a class="anchor-bookmark" id="paste-to-fit-slide"></a> Paste To Fit Slide
+
+**Paste To Fit Slide** helps you paste your copied objects to fit the entire slide.
```

Comment:
Can this be rephrased to clarify how it differs from `Paste to Fill Slide`. The current description sounds like it fills the entire slide, which is not the case. `Fit` here means maximize size without going out of the slide whereas `Fill` means minimum size that covers the entire slide.

Reply:
Hm how about 

>Paste To Fit Slide helps you maximize the size of your copied objects, while staying within the confines of the slide.

Is this acceptable?

Reply:
Something like this? You can rephrase further as you see fit. 
>_Paste To Fit Slide_ pastes the copied object after resizing the object to the maximum size that can fit inside the slide.

Reply:
How about this?

>Paste to Fit Slide helps to resize the object to the maximum size that can fit inside the slide, before pasting the copied object.

Reply:
Updated!